draft-ietf-core-groupcomm-bis.md

```diff
@@ -212,6 +221,8 @@ Further details on identifying a security group are provided in {{sec-groupnamin
 
 If the NoSec mode is used (see {{chap-unsecured-groupcomm}}), group communication does not rely on security at the transport layer nor at the CoAP layer, hence the communicating endpoints do not refer to a security group.
 
+When a security group uses the security protocol Group OSCORE {{I-D.ietf-core-oscore-groupcomm}} to protect group communication (see Section 5), source authentication is achieved for messages exchanged within the group (see {{chap-group-oscore}} and {{chap-security-considerations-sec-mode-sauth}}). That is, even though the endpoints in the security group do share group security material, a recipient CoAP endpoint is able to verify that a message protected with Group OSCORE has actually been originated and sent by a specific and identified CoAP endpoint as a member of the security group.
```

Comment:
Is it Section 5 of this document, or of draft-oscore-groupcomm? That may not be clear after this Markdown is compiled.

Reply:
Indeed it's Section 5 of this document. https://github.com/core-wg/groupcomm-bis/commit/136b0f637b7dc48e296701ad7f556966857436ba  fixes that forward reference and two more in the same sentence.